cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -26,12 +26,12 @@ EOF
   done
 done
 
-gce_cluster_age_cutoff="$(TZ=":Africa/Abidjan" date --date="${CLUSTER_TTL}-4 hours" '+%Y-%m-%dT%H:%M+0000')"
+gce_cluster_age_cutoff="$(TZ=":America/Los_Angeles" date --date="${CLUSTER_TTL}-4 hours" '+%Y-%m-%dT%H:%M%z')"
```

Comment:
I think, keep the `Africa/Abidjan` for UTC stdout, and let the A%z vs B%z work for us.